pkg/distributors/contracts/MerkleRedeem.sol

```diff
@@ -40,6 +40,8 @@ contract MerkleRedeem is IDistributor, Ownable {
 
     IVault public immutable vault;
 
+    event RewardAdded(address indexed token, uint256 amount);
```

Comment:
It seems this new event is not emitted anywhere?

Reply:
This event was moved from `IDistributor` to `MultiRewards` because the addition of the `token` parameter doesn't apply to MerkleRedeem and is used at the end of `notifyRewardAmount`

The parameter would likely be a good addition to the orchard's events, and even the 1-token `MerkleRedeem` so it might find itself back into the `IDistributor` but for now I left `MerkleRedeem.sol` as-is.

Reply:
Ahh I see, thanks